src/libsyntax/feature_gate/active.rs

```diff
@@ -519,6 +519,9 @@ declare_features! (
     /// Allows the use of or-patterns (e.g., `0 | 1`).
     (active, or_patterns, "1.38.0", Some(54883), None),
 
+    /// Enable accurate caller location reporting during panic (RFC 2091).
+    (active, track_caller, "1.39.0", Some(47809), None),
```

Comment:
```suggestion
    (active, track_caller, "1.40.0", Some(47809), None),
```
Missed this before =P